test/interpreter/function-stdevs.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function STDEV.S', () => {
+  it('should take at least two arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=STDEV.S()'],
+      ['=STDEV.S(1)']
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqual(detailedError(ErrorType.DIV_BY_ZERO))
+  })
+
+  it('should calculate standard deviation (sample)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=STDEV.S(2, 3)'],
+      ['=STDEV.S(B2:D2, E2, F2)', 2, 3, 'foo', 'bar', 4],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(0.707106781186548, 6)
+    expect(engine.getCellValue(adr('A2'))).toEqual(1)
+  })
+})
```

Comment:
Missing tests:

- Arguments can either be numbers or names, arrays, or references that contain numbers.
- Logical values and text representations of numbers that you type directly into the list of arguments are counted.
- If an argument is an array or reference, only numbers in that array or reference are counted. Empty cells, logical values, text, or error values in the array or reference are ignored.
- Arguments that are error values or text that cannot be translated into numbers cause errors.
- If you want to include logical values and text representations of numbers in a reference as part of the calculation, use the STDEVA function.

https://support.microsoft.com/en-us/office/stdev-s-function-7d69cf97-0c1f-4acf-be27-f3e83904cc23